firehose.go

```diff
@@ -143,6 +143,6 @@ func (p *Producer) sendFirehoseRecords(args *gokinesis.RequestArgs) {
 	}
 
 	if conf.Debug.Verbose && p.getMsgCount()%100 == 0 {
-		log.Println("Messages sent so far: " + strconv.Itoa(p.getMsgCount()))
+		log.Println("Messages sent so far: " + strconv.FormatInt(p.getMsgCount(), 10))
```

Comment:
We could optionally force the `int64` back to `int` in `getErrCount` and `getMsgCount` but I'm not sure if we will get precision loss errors/warnings.